### PR TITLE
update specs to satisfy AnyInstance requirement

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
Update various files to pass `AnyInstance` cop

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19828

## Testing done
Ran `bundle exec rake ci` to confirm no rubucop offenses are generated. 

## Acceptance criteria
- [x] No errors thrown when running `bundle exec rake ci`

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: spec/models/gibct_site_mapper_spec.rb [hunk 1]
```diff
@@ -15,46 +15,32 @@
...
-        end
-
-        expect(SitemapGenerator::Sitemap.default_host).to eq('https://www.va.gov/gi-bill-comparison-tool')
-
-        sitemap = Zlib::GzipReader.open(sitemaps_path) do |gz|
-          sitemap = gz.read
-          expect(sitemap).to match(Regexp.new("/profile/#{production_institution_fc}"))
-          expect(sitemap).not_to match(Regexp.new("/profile/#{preview_institution_fc}"))
-        end
+    it 'sets the default host' do
```
Comment (shawkey-daniel) [inline spec/models/gibct_site_mapper_spec.rb]: This statement does not appear to match the test.

File: spec/models/gibct_site_mapper_spec.rb [hunk 2]
```diff
@@ -15,46 +15,32 @@
...
     end
 
-    context 'and ping is false' do
-      it 'does not ping the search engines' do
-        expect_any_instance_of(described_class).not_to receive(:ping_search_engines)
-
-        SiteMapperHelper.silence do
-          described_class.new(false)
-        end
+    it 'sets SitemapGenerator::Sitemap.default_host' do
```
Comment (shawkey-daniel) [inline spec/models/gibct_site_mapper_spec.rb]: This statement is not in the correct natural language and does not clearly match the test.

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: spec/models/gibct_site_mapper_spec.rb [hunk 3]
```diff
@@ -15,47 +15,36 @@
...
   end
 
   describe 'when initializing' do
-    it 'sets the default host and creates a sitemap with only production data' do
-      [true, false].each do |ping|
-        SiteMapperHelper.silence do
-          mapper = described_class.new(ping)
-          expect(mapper.sitemap_location).to eq('https://www.va.gov/gids/sitemap.xml.gz')
-        end
+    it 'sets the default host and creates a sitemap' do
```
Comment (shawkey-daniel) [inline spec/models/gibct_site_mapper_spec.rb]: This one is checking the sitemap URI

File: spec/models/gibct_site_mapper_spec.rb [hunk 4]
```diff
@@ -15,47 +15,36 @@
...
+      SiteMapperHelper.silence do
+        mapper = described_class.new(false)
+        expect(mapper.sitemap_location).to eq('https://www.va.gov/gids/sitemap.xml.gz')
+      end
+    end
 
+    it 'sets default_host to www.va.gov/gi-bill-comparison-tool' do
```
Comment (shawkey-daniel) [inline spec/models/gibct_site_mapper_spec.rb]: This one is checking the default host URI

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by LindseySaari — APPROVED** [review]
👍

**Pull request merged**